Translations/bg.po

```diff
@@ -4678,19 +4868,19 @@ msgstr ""
 
 #: Source/monstdat.cpp:488
 msgid "Pukerat the Unclean"
-msgstr ""
+msgstr "Поквареният Пукерат"
 
 #: Source/monstdat.cpp:489
 msgid "Boneripper"
-msgstr ""
+msgstr "Костотрошача"
 
 #: Source/monstdat.cpp:490
 msgid "Rotfeast the Hungry"
 msgstr ""
 
 #: Source/monstdat.cpp:491
 msgid "Gutshank the Quick"
-msgstr ""
+msgstr "Светкавицата Гътшанк"
```

Comment:
Maybe not all names should be translated, still this guy shouldn't be that fast, instead of "Светкавицата" I´d go for "Пъргавия".  And for Gutshank maybe "Мършочрев" or something in that spirit. Using Bulgarian words to construct the name.

Reply:
```suggestion
msgstr "Пъргавия Чреволеш"
```

Reply:
Agreed.